contract/models/contract_recurrency_mixin.py

```diff
@@ -89,7 +94,7 @@ class ContractRecurrencyMixin(models.AbstractModel):
         string="Next Period End",
         compute="_compute_next_period_date_end",
     )
-    last_date_invoiced = fields.Date(readonly=True, copy=False)
+    last_date_invoiced = fields.Date(copy=False)
```

Comment:
Don't remove readonly, which is unrelated to the goal of this PR.

Reply:
Changed